incubator/tensorflow-inception/values.yaml

```diff
@@ -11,10 +11,6 @@ Replicas: 1
 Image: "quay.io/lachie83/inception_serving"
 ImageTag: "latest"
 ImagePullPolicy: "IfNotPresent"
-Resources:
-  limits:
-    cpu: 100m
-    memory: 256Mi
-  requests:
-    cpu: 100m
-    memory: 256Mi
\ No newline at end of file
+Cpu: 100m
+Memory: 256Mi
+Gpu: 0
```

Comment:
Although this works fine, it seems that we're moving to standardising on the `resources` variable for this instead, which just takes freestyle yaml so someone is able to set the resources like normal. I think that's cleaner, although adding the cpu option to the docs and/or the values.yaml example might be good as well.